parsers/FR_O.py

```diff
@@ -230,17 +238,18 @@ def get_date_name(zone_key: None, target_datetime: None):
 # Non-thermal sources names in api
 sources_mapping = {
     'biomass': ['bioenergies_mwh', 'bioenergies'],
-    'hydro': ['hydraulique_mwh', 'hydraulique'],
-    'solar': ['photovoltaique_mwh', 'photovoltaique', 'photovoltaique_avec_stockage'],
+    'hydro': ['hydraulique_mwh', 'hydraulique', 'micro_hydro'],
+    'solar': ['photovoltaique_mwh', 'photovoltaique', 'photovoltaique_avec_stockage', 'photovoltaique0'],
     'wind': ['eolien_mwh', 'eolien'],
     'geothermal': ['geothermie_mwh'],
-    'gas': ['biogaz']
+    'gas': ['biogaz'],
```

Comment:
Didn't want to change this before double checking but `biogaz` should be mapped to `biomass` right?

Reply:
I think so, right @pierresegonne?

Reply:
Correct